packages/main/src/Dialog.ts

```diff
@@ -79,6 +79,25 @@ const ICON_PER_STATE: Record<ValueStateWithIcon, string> = {
  * <br>
  * For more information see the sample "Bar in Header/Footer".
 
+ *<h3>Keyboard Handling</h3>
+ *
+ * <h4>Basic Navigation</h4>
+ * When the <code>ui5-dialog</code> is focused and has the <code>draggable</code> property set, the user can move the dialog
```

Comment:
The focus  should be on the header in order to use the keys to drag the dialog.
I suggest:
When the header of the code>ui5-dialog</code> is focused.

But in this case we should say first that the dialog is draggable (if its not - the header can't be focused :) ).
So this:
"When the <code>ui5-dialog</code> is focused and has the <code>draggable</code> property set"
could be said as this:
"When the <code>ui5-dialog</code> has the <code>draggable</code> property set and the header is focused"